templater/jobs/presubmit/eks-distro-build-tooling/prow-plugin-presubmit.yaml

```diff
@@ -11,6 +11,3 @@ resources:
   requests:
     cpu: 2560m
     memory: 16Gi
-envVars:
-  - name: PUSH_IMAGES
-    value: false
```

Comment:
I think we still want `PUSH_IMAGES` to be `false` in presubmit, right?

Reply:
Is that not done through the local-image target?